### ISSUE TITLE
Updating text attributes doesn't work properly

### ISSUE THREAD

**Issue by mVermaat** [Microsoft/EasyRepro#375]
When updating a text field on a form that already has a value, the value disappears after entering the data.

It can be reproduced by performing the following actions:

1) Create new account with Account name 'Test Account'
2) Save & Close account
3) Reopen account
4) Change account name to 'Updated'

Before update:
![image](https://user-images.githubusercontent.com/13309338/52637833-b8cc9000-2ed0-11e9-8444-3b3613de29c0.png)

After calling 'xrmBrowser.Entity.SetValue("name", "Updated");':
![image](https://user-images.githubusercontent.com/13309338/52637877-def23000-2ed0-11e9-9180-d1a4fd168dea.png)

Now the field looks empty. If i would after click the save button again, or manually click on the field the data appears (see below). It seems the data is entered, but it isn't properly registered by Dynamics. 

![image](https://user-images.githubusercontent.com/13309338/52637912-faf5d180-2ed0-11e9-8fd7-07e9322021bb.png)

**Comment by fatboyspy** [comment]
Could be solved by using a small crutch, like that

```
xrmBrowser.Entity.SetValue("name", "");
xrmBrowser.Entity.SetValue("name", "John Smith (Test API) updated");
xrmBrowser.Entity.SetValue("telephone1", "");
xrmBrowser.Entity.SetValue("telephone1", "445-445-5555");
xrmBrowser.Entity.SetValue("fax", "");
xrmBrowser.Entity.SetValue("fax", "777-555-333");
xrmBrowser.Entity.SetValue("websiteurl","");
xrmBrowser.Entity.SetValue("websiteurl", "https://google.com");
```

Helps in my case

**Comment by jlattimer** [comment]
Merged into 9.1 branch - closing
